sycl/test/basic_tests/sampler/sampler.cpp

```diff
@@ -1,3 +1,5 @@
+// REQUIRES: opencl
+
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -L %opencl_libs_dir -lOpenCL
 // RUN: env SYCL_DEVICE_TYPE=HOST %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
Should we remove lines 9 and 10?
```c++
// TODO: Image support in CUDA backend
// XFAIL: cuda
```

Reply:
It would not hurt though might be better as an extra PR to not block this one that re-enables LIT testing with CUDA?

Reply:
I'm okay to resolve this comment in a separate pull request.
@bjoernknafla, will do this?
We need to clean-up following files from this PR:
```
basic_tests/buffer/buffer.cpp
basic_tests/sampler/sampler.cpp
fpga_tests/fpga_queue.cpp
ordered_queue/ordered_buffs.cpp
ordered_queue/ordered_dmemll.cpp
sub_group/common_ocl.cpp
```
I think all these files should have something like:
```c++
// REQUIRES: opencl
// UNSUPPORTED: cuda
```

Reply:
If I remember correctly @bjoernknafla mentioned that XFAIL: cuda was there to indicate that the test fails for CUDA and needs to be fixed. Have all the TODOs been resolved for ^ tests?

Reply:
These are requiring OpenCL, so I assume that none of them can be "fixed on CUDA". Am I missing something?

Reply:
`sampler.cpp` should have a `UNSUPPORTED: cuda` on it.
If the others already have `REQUIRES: opencl` nothing else should be required. However, I will look over this in an upcoming PR, so we could get LIT fixes of this PR in.